base/utf8proc.jl

```diff
@@ -75,10 +75,7 @@ function utf8proc_map(s::ByteString, flags::Integer)
                    s, sizeof(s), p, flags)
     result < 0 && error(bytestring(ccall(:utf8proc_errmsg, Ptr{UInt8},
                                          (Cssize_t,), result)))
-    a = ccall(:jl_ptr_to_array_1d, Vector{UInt8},
-              (Any, Ptr{UInt8}, Csize_t, Cint),
-              Vector{UInt8}, p[], result, true)
-    ccall(:jl_array_to_string, Any, (Any,), a)::ByteString
+    pointer_to_string(p[], result, true)::ByteString
```

Comment:
No, it's not unrelated.  What happened is that I needed exactly the same code in FFTW.jl, and rather than copy-and-paste the code it seemed better to put it into a function, analogous to `pointer_to_array`.